content/logs/log_collection/_index.md

```diff
@@ -237,19 +237,21 @@ logs:
 
 ### Using a Proxy for Logs
 
-The log Agent does not presently respect the usual proxy setting in the `datadog.yaml` configuration file. It is **not yet** possible to use the agent as proxy (local agent -> proxy agent -> Datadog) but a third party proxy can be used.
+The log Agent does not presently respect the usual proxy setting in the `datadog.yaml` configuration file, since logs are only forwarded through TCP/SSL and not HTTPs.
```

Comment:
- consistent caps please.  "log Agent"
- Also Datadog has one Datadog Agent. Logs, Traces, Metrics are Processes are features it provides. Lets avoid language that implies theres a separate log agent package.

Reply:
"Logs does does not presently respect the global proxy settings in the `datadog.yaml` configuration file. This is due to logs presently being transported over TCP/SSL while other Datadog features rely on HTTPS."

OR better yet:

"Logs make use of a different set of proxy settings than other data types forwarded by the Datadog Agent.  This is due to logs presently being transported over TCP/SSL, while other features submit data via on HTTPS."


Do we need to call out environment variables in addition to datadog.yaml?